scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -147,24 +140,27 @@
 
 {% macro panel_view(institute, panels) %}
   <div class="card panel-default w-100 mt-3 justify-content-center">
-    <div class="panel-heading">
-      <strong>{{ institute.display_name }}</strong> - Panels
+    <div class="card-title d-flex justify-content-between">
+      <span><strong>{{ institute.display_name }}</strong> - Panels</span>
+      <span>
+        <input type="checkbox" name="hideLine" checked onclick="toggleRow(this)"> Hide removed panels</input
+      </span>
     </div>
     <div class="card-body">
-        <table class="table table-striped">
+        <table class="table table-striped" id="panelTable" style="table-layout: fixed;"">
         <thead>
           <tr>
-            <th width="30%">Name</th>
-            <th width="20%">Version</th>
-            <th width="15%">Number of genes</th>
-            <th width="25%">History</th>
-            <th width="10%"></th>
+            <th>Name</th>
+            <th>Version</th>
+            <th>Number of genes</th>
+            <th>History</th>
+            <th></th>
           </tr>
         </thead>
       {% for panel in panels|sort(attribute="display_name",case_sensitive=False) %}
         <!--create table for each panel-->
         <tbody>
-          <tr>
+          <tr {% if panel.hidden %} class="hidableRow" style="visibility:collapse;" {% endif %}>
```

Comment:
I would have used hid_**e**_ableRow instead, but I guess if it is already hidden, it was also hidable? 😊

Reply:
Toh I found it also in the famous merriam webster dictionary: https://www.merriam-webster.com/dictionary/hidable 🤣